charts/logzio-k8s-events/values.yaml

```diff
@@ -0,0 +1,70 @@
+# Default values for logzio-k8s-events.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+image:
+  repository: ralatdocker/logzio-k8s-events
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "0.0.1"
+
+k8sApp: logzio-k8s-events
+namespace: monitoring
```

Comment:
It's better to use {{ .Release.Namespace }} instead of allowing to set it elsewhere. This way, you make sure that there's a match between the namespace of the Chart itself and the namespace of the resources it created.
You can see an example here
https://github.com/logzio/logzio-helm/blob/a0b22fdfc10c229d5f17f94b264e727bd53a2a94/charts/fluentd/templates/clusterrole.yaml#L5